openqabot/amqp.py

```diff
@@ -60,9 +60,9 @@ def stop(self) -> None:
 
     def on_message(
         self,
-        unused_channel: pika.channel.Channel,  # noqa: ARG002 Unused method argument
+        _: pika.channel.Channel,
         method: pika.spec.Basic.Deliver,
-        unused_properties: pika.spec.BasicProperties,  # noqa: ARG002 Unused method argument
+        __: pika.spec.BasicProperties,
```

Comment:
seems like double underscore here. Can you take a look?

Reply:
That's on purpose to mark the unused method arguments. I can't use `_` two times